Makefile

```diff
@@ -8,4 +8,4 @@ install: all
 	  bn=$$(basename $$x); \
 	  install -D -t $(DESTDIR)/usr/lib/dracut/modules.d/$${bn} $$x/*; \
 	done
-	install -D -m 644 -t $(DESTDIR)/usr/lib/systemd/system systemd/*
+	install -D -m 644 -t $(DESTDIR)/usr/lib/systemd/system systemd/system/*
```

Comment:
This should go in the previous commit.  But since this PR is no longer adding generators, should we drop this and the rename to ease packaging?

Reply:
I originally left it when dropping the generator just because it lined up better with the destination but dropping it until we need to add something to a different systemd folder works for me.